.github/scripts/build-linux.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
```

Comment:
agreed, not sure why they were named this and also why they aren't in `scripts` folder as that would make it usable/discoverable for others to consume too.

Reply:
I think they are in `.github/scripts` instead of plain `scripts` because they were intended to be used in github workflows.
If we don't plan to use them outside of workflows, I think the current folder is fine.

Reply:
we give this set of instructions in the [readme](https://github.com/containerd/runwasi#building), we could put it behind a make file target and use it locally and in ci which would keep them in sync